pkg/crc/machine/start.go

```diff
@@ -524,6 +524,10 @@ func createHost(api libmachine.API, machineConfig config.MachineConfig) error {
 		return fmt.Errorf("Error in driver during machine creation: %s", err)
 	}
 
+	logging.Info("Generating new SSH Key pair...")
+	if err := crcssh.GenerateSSHKey(constants.GetPrivateKeyPath()); err != nil {
```

Comment:
Can you add a few sentences to the commit log explaining why this ssh change is needed? As it is now, it's just code being moved around and a bit simpler code, not sure more than that is expected.

Reply:
I added more context. 
Yes this is just a code move. The goal is to have the machine directory ready before the machine is marked as existing.